labelbox/schema/project.py

```diff
@@ -622,9 +631,12 @@ def create_batch(self,
                                   params,
                                   timeout=180.0,
                                   experimental=True)["project"][method]
-
-        res['size'] = len(dr_ids)
-        return Entity.Batch(self.client, self.uid, res)
+        batch = res['batch']
+        batch['size'] = len(dr_ids)
+        return Entity.Batch(self.client,
+                            self.uid,
+                            batch,
+                            failed_data_row_ids=res['failedDataRowIds'])
```

Comment:
What happens if this is 100k or more data row ids?

Reply:
We don't allow more than 20K of data rows to be specified for a batch so here it can be 20K ids at most. That's about 500KB of data